ui/src/AudioRecorder.js

```diff
@@ -1,44 +1,48 @@
 export class AudioRecorder {
     /**
-    * Creates an AudioRecorder that can record utterances
-    * @param {String} stores and allows access to utterance after recording
-    * @param {Object} allows access to Web Audio API
-    */
+     * Creates an AudioRecorder that can record utterances
+     * @param {String} recordingUrl - A URL that allows access to an audio file after recording.
+     * @param {Blob} blob - An audio recording file. 
+     * @param {Object} mediaRecorder - Allows access to Web Audio API.
+     */
     constructor() {
         this.recordingUrl;
+        this.blob;
         this.mediaRecorder;
     }
+
     /**
-    * Prompts user for access to Microphone using API and starts recording is access granted
-    */
+     * Prompts user for access to Microphone Component and begins recording if access is granted.
```

Comment:
Does it ask the permission every time they hit the button?

Reply:
It will ask for permission the first time the visit the page and press the button. After, it will never ask permission again unless the Microphone permissions are updated in Chrome settings